### PR TITLE
Accesibility:Milestone 6

### PR DESCRIPTION
**Pull request by Tumworobere**
**In this milestone, We  did the following:**
- Pages title :  [link to the code](https://github.com/Tumworobere/Portfolio/blob/ed7ec1c1f893a42b79216f2cdc59ddcc80b51108/index.html#L17)
- Images text alternative : [link to the code](https://github.com/Tumworobere/Portfolio/blob/ed7ec1c1f893a42b79216f2cdc59ddcc80b51108/index.html#L52-L56)
- Text headings: no issues found
- Color contrast: [link to the code](https://github.com/Tumworobere/Portfolio/blob/ed7ec1c1f893a42b79216f2cdc59ddcc80b51108/style.css#L436-L443)
- Resize : [link to the code](https://github.com/Tumworobere/Portfolio/blob/ed7ec1c1f893a42b79216f2cdc59ddcc80b51108/style.css#L605-L610)
- Interaction: No issues were found 
- Moving content: No issues found
- Multimedia: No issues found
- Basic structure of the page: No issues were found

### CODE REVIEW

**Review by DanSam5K — CHANGES_REQUESTED** [review]
## Hi  @Tumworobere  and  @RolandM99,  ☺️ 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that linter check is not passing, I would advise you fixed the linter error and request another review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
   gtag('config', 'G-1NWRQ1EMTE');
```
Comment (SajjadAhmad14) [inline index.html:15]: Please add the accessibility report in Pull Request.

The report should include a check about page titles.
The report should include a check about image text alternatives (alt="alt text ").
The report should include a check about text headings.
The report should include a check about color contrast.
The report should include a check about resize.
The report should include a check about interaction.
The report should include a check about moving content.
The report should incldue a check about multimedia.
The report should include a check about the basic structure of the page.

If everything is ok, please make a statement about that (e.g. "Page titles - no issues found").
If issues are found, please add  link to the corresponding line of code in the Pull Request changes

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Tumworobere, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -245,17 +246,17 @@ <h2>Profesional Art Printing Data</h2>
             </section>
             <section class="about">
```
Comment (SajjadAhmad14) [inline index.html:247]: Some buttons have failed the button contrast tests. You can use this site to test it. Kindly see the screenshot below for reference
contrast
.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Tumworobere, 
 ### Thank you for implementing the changes suggested. :ok_hand::
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (RolandM99) [reply index.html:247]: image

As we can see above, we solved the failed contrast test yet

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Tumworobere, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**